### PR TITLE
Update archlinux PKGBUILD-git.

### PR DESCRIPTION
**Pull request by streetsamurai00mi**
Remove unnecessary prerare function.
Modify PKGBUILD-git in order to comply with new install script. See 4d1f713.

### CODE REVIEW

File: build/arch/PKGBUILD-git [hunk 1]
```diff
@@ -44,5 +41,5 @@ build() {
 package() {
   cd "$pkgname"
 
-  make DESTDIR="$pkgdir/" install
+  make PREFIX="/usr" DESTDIR="$pkgdir/" install
```
Comment (goatastronaut0212) [inline build/arch/PKGBUILD-git:44]: I wonder why you put PREFIX to Makefile? Have you tested it? It doesn't work on my Arch machine

**Review by goatastronaut0212 — CHANGES_REQUESTED** [review]

**Comment by streetsamurai00mi** [comment]
> why you put PREFIX to make command

https://github.com/BambooEngine/ibus-bamboo/blob/4d1f71357d12c7ab3634b0f74564253da91c8ebb/Makefile#L54-L55
https://github.com/BambooEngine/ibus-bamboo/commit/4d1f71357d12c7ab3634b0f74564253da91c8ebb#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R55


> Have you tested it?

yes, and pushed to aur https://aur.archlinux.org/packages/ibus-bamboo-git#comment-991486


> It doesn't work on my Arch machine

Have you tried `-f, --force` (Overwrite existing package) and `-C, --cleanbuild` (Remove $srcdir/ dir before building the package)

**Comment by streetsamurai00mi** [comment]
Signed the commit and pushed to remote repo using force flag.

**Comment by goatastronaut0212** [comment]
I spend more time to test again. Seem pretty okay. I guest I will merge it. Thank for your contribution @streetsamurai00mi

**Pull request merged**